client/store/user.js

```diff
@@ -39,6 +39,19 @@ export const auth = (email, password, method) =>
       })
       .catch(dispatchOrHistoryErr => console.error(dispatchOrHistoryErr));
 
+export const signUpGuest = (firstName, lastName, password) => {
+  return dispatch => {
+    return axios.post('/auth/guestsignup', { firstName, lastName, password })
+      .then(res => {
+        return dispatch(getUser(res.data));
+      })
+      // }, authError => { // rare example: a good use case for parallel (non-catch) error handler
+      //   dispatch(getUser({ error: authError })); // I don't think that we need these, but probably
+      // })                  // Something good to go over in the code review
```

Comment:
Yeah, we can go over it in code review. I think though that we shouldn't have these comments in master.